packages/kbn-test/src/jest/setup/react_testing_library.js

```diff
@@ -43,7 +43,9 @@ jest.mock('@testing-library/react', () => {
 const originalConsoleError = console.error;
 console.error = (...args) => {
   if (global.IS_REACT_ACT_ENVIRONMENT === false) {
-    if (args[0].includes('Warning: An update to %s inside a test was not wrapped in act')) {
+    if (
+      args[0].toString().includes('Warning: An update to %s inside a test was not wrapped in act')
```

Comment:
I had to convert to String as the error that was passed was an "object" type that does not have the `includes` prop